### DISCUSSION TITLE
Metavoice Discord Issues?

### DISCUSSION THREAD

**Discussion by addictivepixels** [metavoiceio/metavoice-src#57] [General]
Myself and another person are unable to successfully agree to the Discord's rules - agreeing results in "This interaction failed".

I can't message the mods due to privacy restrictions - wasn't sure if anyone else is having this issue or if anyone (hopefully a Mod 😄) can relay this to their mod/admin team.

Sidenote, I forked the project and was successfully able to get it running in Windows 😄

**Comment by sidroopdaska** [comment]
Thanks for flagging this. I'm on it.

**Comment by Promootheus** [comment]
Also having the same issue, made sure to install the latest and still cannot Agree .

**Comment by sidroopdaska** [comment]
Here's the new invite link: https://discord.gg/tbTbkGEgJM
